pydantic_ai_slim/pydantic_ai/_result.py

```diff
@@ -29,25 +29,22 @@ def __post_init__(self):
     async def validate(
         self,
         result: ResultData,
-        deps: AgentDeps,
-        retry: int,
         tool_call: _messages.ToolCallPart | None,
-        messages: list[_messages.ModelMessage],
+        run_ctx: RunContext[AgentDeps],
```

Comment:
We call this `ctx` in our examples, should we be consistent and to that here?

Reply:
Or maybe we could just fully use `run_context`?

Reply:
yes, probably `run_context` internally is good.